src/tyr/locate_serializer.cc

```diff
@@ -1,11 +1,57 @@
 #include "baldr/json.h"
+#include "baldr/openlr.h"
 #include "tyr/serializers.h"
 #include <cstdint>
 
 using namespace valhalla;
+using namespace valhalla::midgard;
 using namespace valhalla::baldr;
 
 namespace {
+
+OpenLR::LocationReferencePoint::FormOfWay get_fow(const baldr::DirectedEdge* de) {
```

Comment:
i moved the patched versions of these functions over to the serializer where they are used. ive also converted the tests to check them after serialization to get the full round trip whilest also avoiding polluting the public interface with what are implementation details.